From 917d914a46c7344b6a63aca5152507e2cff96d68 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Wed, 8 Aug 2018 09:02:33 +0800
Subject: [PATCH 4355/5242] MLK-19153-1 ARM: dts: update new "fsl,tx-d-cal"
 property

commit  c3c54434d655bb56049bbaaeefcc61fb0f5bf6fc from
https://source.codeaurora.org/external/imx/linux-imx.git

Using new "fsl,tx-d-cal" for dts, and update document accordingly.

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/phy/mxs-usb-phy.txt        |    5 +----
 arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi         |    4 ++--
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi           |    6 ------
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi             |    4 ++--
 arch/arm/boot/dts/imx6sl-evk.dts                   |    4 ++--
 arch/arm/boot/dts/imx6sx-sdb.dts                   |    4 ++--
 arch/arm/boot/dts/imx7ulp-evk.dts                  |    2 +-
 7 files changed, 10 insertions(+), 19 deletions(-)

diff --git a/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt b/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
index 16e2072..fe1a475 100644
--- a/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
+++ b/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
@@ -23,9 +23,6 @@ Optional properties:
   that terminates the DP output signal. Default: 45
 - fsl,tx-d-cal: Integer [79-119]. Current trimming value (as a percentage) of
   the 17.78mA TX reference current. Default: 100
-- tx-d-cal: Try to adjust this value to improve signal quality, and pass
-  USB Certification, the value is from 0x0 to 0xf, and the register offset
-  is 0x10 (USBPHY_TX).
 
 Example:
 usbphy1: usbphy@20c9000 {
@@ -33,5 +30,5 @@ usbphy1: usbphy@20c9000 {
 	reg = <0x020c9000 0x1000>;
 	interrupts = <0 44 0x04>;
 	fsl,anatop = <&anatop>;
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
diff --git a/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi b/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi
index 361fe62..cff0b75 100644
--- a/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi
+++ b/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi
@@ -568,11 +568,11 @@
 };
 
 &usbphy1 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usbphy2 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usdhc2 {
diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index 73c35f0..3dd6674 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -1057,12 +1057,6 @@
 				MX6QDL_PAD_EIM_DA0__EIM_AD00		0xb0b1
 			>;
 		};
-
-		pinctrl_hdmi_cec: hdmicecgrp {
-			fsl,pins = <
-				MX6QDL_PAD_EIM_A25__HDMI_TX_CEC_LINE 0x1f8b0
-			>;
-		};
 	};
 };
 
diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 703040e..4886b0b 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -1089,11 +1089,11 @@
 };
 
 &usbphy1 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usbphy2 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usdhc2 {
diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index aa20735..089b61a 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -912,11 +912,11 @@
 };
 
 &usbphy1 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usbphy2 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usdhc1 {
diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 6ce4db1..ca02fdb 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -83,9 +83,9 @@
 };
 
 &usbphy1 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
 
 &usbphy2 {
-	tx-d-cal = <0x5>;
+	fsl,tx-d-cal = <106>;
 };
diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 4251948..fd760a6 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -537,7 +537,7 @@
 };
 
 &usbphy1 {
-	tx-d-cal = <0xc>;
+	fsl,tx-d-cal = <88>;
 };
 
 &usdhc0 {
-- 
1.7.9.5

